doc/install_mne_c.rst

```diff
@@ -2,290 +2,162 @@
-the environment variable MNE_ROOT.
-
-Your system-dependent path to MATLAB Software will be referred to by
-the environment variable MATLAB_ROOT.
+Configuration
+^^^^^^^^^^^^^
 
-You can download a free runtime distribution of Matlab with the libraries
-necessary to run Matlab code from,
-https://www.mathworks.com/products/compiler/matlab-runtime.html
+MNE-C requires two environment variables to be defined manually:
 
-If you do not have a Matlab distribution, then leave MATLAB_ROOT undefined.
-Please note however that if Matlab is not available, the utilities
-mne_convert_mne_data, mne_epochs2mat, mne_raw2mat, and mne_simu will not work.
+- ``MNE_ROOT`` should give the path to the folder where MNE-C is installed
+- ``MATLAB_ROOT`` should give the path to your MATLAB binary (e.g.,
+  ``/opt/MATLAB/R2018b`` or similar).  If you do not have MATLAB or the MATLAB
+  runtime, leave ``MATLAB_ROOT`` undefined.
 
-To setup the MNE environment, a script must be sourced into your current shell.
-The result will be that additional environment variables like MNE_LIB_PATH
-will be set in your current shell in order to run the MNE Software.
-
-A note about source-ing scripts with "source" and "." (dot). The hash bang at
-the top of a script tells exec what shell to use by default to interpret the
-script. But the hash bang line is ignored when you source a script via either
-the builtin "source" or "." directive.  So there needs to be a setup script
-compatible with your current shell, i.e., if you use csh you cannot source a
-bash script.  (BTW, the "." (dot) directive will search for the script in
-$PATH if there is no slash in the script argument while "source" does not
-search $PATH).
-
-For Bourne or bash compatible shells, e.g., sh/bash/zsh, the script to source
-is ``$MNE_ROOT/bin/mne_setup_sh``.
-
-For C shells, e.g., csh/tcsh, the script to source
-is ``$MNE_ROOT/bin/mne_setup``.
-
-The SHELL variable should already be set in your current shell by your login or
-profile init file.  You can echo $SHELL on the command line to see what type of
-shell you are using with:
+Other environment variables are defined by setup scripts provided with MNE-C.
+You may either run the setup script each time you use MNE-C, or (recommended)
+configure your shell to run it automatically each time you open a terminal. For
+bash compatible shells, e.g., sh/bash/zsh, the script to source is
+``$MNE_ROOT/bin/mne_setup_sh``.  For C shells, e.g., csh/tcsh, the script to
+source is ``$MNE_ROOT/bin/mne_setup``.  If you don't know what shell you are
+using, you can run the following command to find out:
 
 .. code-block:: console
 
     $ echo $SHELL
 
-If the output indicates a POSIX shell ``bash`` or ``sh`` then you should export
-variables and source the bash/sh setup file with three commands:
+To configure MNE-C automatically for ``bash`` or ``sh`` shells, add this to
+your ``.bashrc``::
 
-.. code-block:: console
```

Comment:
reason for doing this was that we're telling people to put these into their `.bashrc` (or `.zshrc`, etc), rather than type into their interactive shell... I thought people might wrongly copy-paste the leading `$`.  But I can put it back if you think it actually helps.

Reply:
Then it would be `code-block:: bash`. The default linter is `Python` for `::` (and IIRC it just decides "is Python" or "is not Python", in this case it will do the latter which is not as good as "is Bash", which the `code-block` will do).

But FWIW we used CSS to disable the `$` copying, so they physically can't do it anymore. So I'm fine with `code-block:: bash` or `code-block:: console`.

Reply:
Really?  I can select-copy-paste the `$` on this page of the current `dev` docs: https://mne-tools.github.io/dev/install_mne_python.html

Either way, `code-block:: bash` is better than what I've currently done.

Reply:
Here is my CTRL-A (same thing when using mouse):

![screenshot from 2019-02-08 16-08-59](https://user-images.githubusercontent.com/2365790/52506281-10f74f80-2bbc-11e9-8ea2-8a4815943d52.png)

What browser are you using? Maybe it does not respect `user-select` in CSS. Or maybe you need to hard-reload (shift-F5) the page to get it to reload all resources, including CSS files.

Reply:
Here is my Ctrl+A after force-reload of both `style.css` (which I opened in its own tab for that purpose) and of the page in question.

![mne-select-all](https://user-images.githubusercontent.com/1810515/52506702-d1b90680-2b9b-11e9-9552-575ce4e636a5.png)

Browser is firefox 65.0 for Linux...  I did an `apt upgrade` yesterday so it's as current as they come.

Reply:
Fix in #5911